packages/block-library/src/video/edit.js

```diff
@@ -150,22 +158,37 @@ class VideoEdit extends Component {
 			this.setState( { src: media.url, editing: false } );
 		};
 
+		const editImageIcon = ( <SVG width={ 20 } height={ 20 } viewBox="0 0 20 20"><Rect x={ 11 } y={ 3 } width={ 7 } height={ 5 } rx={ 1 } /><Rect x={ 2 } y={ 12 } width={ 7 } height={ 5 } rx={ 1 } /><Path d="M13,12h1a3,3,0,0,1-3,3v2a5,5,0,0,0,5-5h1L15,9Z" /><Path d="M4,8H3l2,3L7,8H6A3,3,0,0,1,9,5V3A5,5,0,0,0,4,8Z" /></SVG> );
```

Comment:
Isn’t it the same icon duplicated in all components?

Reply:
:( yes it is :D that's the kind of stuff I put WIP in the title for. On the other hand, in this particular case, these all are in the block-library package, where do you think is the most appropriate place to put this icon so it's accessible to all blocks?

Reply:
@jasmussen - is it something that should be moved to `Dashicons` component? I think this is what we did in the past several times.

Reply:
I believe the Dashicons component Gutenberg uses [comes directly from](https://github.com/WordPress/dashicons/blob/master/react/index.jsx) the Dashicons repo, which would mean we'd need to merge this icon there first. There've been lots of discussions around abandoning Dashicons in core, so I'm not really sure if that's what we want here. 🤔 Interested to hear what Joen thinks. 

Totally agree that we should put this in a central place though. Doesn't make sense to duplicate it so frequently.

Reply:
Apologize for missing this ping. I feel like Github notifications could do with some solid improvements, so actual mentions are unmissable.

The Dashicons component is created and maintained in a separate repo, and icons added there get a sanity check and are then merged and we copy the component here. It's not a great flow, and as Kjell mentioned we are looking at ways to smooth out this flow. 

In the mean time, I don't think we need to add this icon to Dashicons just to use it here. I think it's high time we look at a better flow, and I'm not sure it helps Dashicons to be overloaded with additional icons in the mean time. This is a personal opinion, but it's why I would suggest this icon shouldn't be a blocker.